malcolm/modules/pmac/parts/pmactrajectorypart.py

```diff
@@ -1,11 +1,14 @@
 import numpy as np
 from annotypes import add_call_types, Anno, Array
 
-from malcolm.core import PartRegistrar, BooleanMeta, config_tag, Widget, \
-    NumberMeta
+from malcolm.core import PartRegistrar, Widget, \
```

Comment:
Can you put IncompatibleError in malcolm.core.__init__ and import it from there please?

Reply:
Done